vaadin-testbench-core/pom.xml

```diff
@@ -378,28 +378,53 @@
             <artifactId>kotlin-stdlib</artifactId>
             <version>${kotlin.version}</version>
         </dependency>
+        <dependency>
+            <groupId>org.jetbrains.kotlin</groupId>
+            <artifactId>kotlin-stdlib-jdk8</artifactId>
```

Comment:
kotlin-stdlib-jdk8 brings kotlin-stdlib with it so the kotlin-stdlib can be removed from dependencies

Reply:
done